src/main/scala/org/zalando/spearheads/innkeeper/dao/Audits.scala

```diff
@@ -0,0 +1,49 @@
+package org.zalando.spearheads.innkeeper.dao
+
+import java.time.LocalDateTime
+
+import org.zalando.spearheads.innkeeper.dao.MyPostgresDriver.api._
+
+object Audits extends TableQuery(new AuditsTable(_))
+
+case class AuditRow(
+  id: Option[Long] = None,
+  userId: String,
+  resource: String,
+  resourceId: Long,
+  auditType: String,
+  timestamp: LocalDateTime,
+  entity: String)
+
+class AuditsTable(tag: Tag) extends Table[AuditRow](tag, "AUDITS") {
+  def id = column[Long]("AUDIT_ID", O.PrimaryKey, O.AutoInc)
+  def userId = column[String]("USER_ID")
+  def resource = column[String]("RESOURCE")
+  def resourceId = column[Long]("RESOURCE_ID")
+  def auditType = column[String]("AUDIT_TYPE") // create, update, delete
```

Comment:
I would leave it like this for now. The Audits table is write only at the moment and that should be done via the repository. This mapping is being done there. The outside world expresses its intent via the AuditType sealed trait. If it's OK with you, I would add the extra complexity when it becomes required.